core/src/klio_core/config/_io.py

```diff
@@ -199,7 +214,7 @@ def to_io_kwargs(self):
 
 @attr.attrs(frozen=True)
 class KlioPubSubConfig(object):
-    name = "pubsub"
+    type_name = "pubsub"
```

Comment:
I think we should use `TYPE_NAME` in caps to indicate this is more like a class-level constant.